### DISCUSSION TITLE
Using emoji markup vs actual unicode?

### DISCUSSION THREAD

**Discussion by astrochun** [Textualize/rich#2191] [Q&A] [answered]
I really like the support for emoji and have a general question.

Is there an advantage to using the markup over the emoji themselves:
For example `:heavy_check_mark:` vs ✔

I find that the markup extends my code beyond 79 characters. The latter seems to work but I wonder if there are certain terminals and file logging this would fail that `rich` addresses.

Thanks!

**Comment by willmcgugan** [comment, marked as answer]
The emoji code is a convenience, but its fine to replace it with the actual character. No difference as far as Rich is concerned.

**Reply by astrochun** [reply]
Thanks for the quick response.
